assets/src/components/pr/automations/prConfigurationUtils.tsx

```diff
@@ -49,16 +49,17 @@ export function validateAndFilterConfig(
 
     if (conditionMet && name) {
       let value: string | boolean | undefined = configVals[name]
+      const isBool = configItem.type === ConfigurationType.Bool
 
-      if (configItem.type === ConfigurationType.Bool) {
+      if (isBool) {
         value = parseToBool(value)
       }
 
-      if (value) {
-        filteredValues.push({ name, value })
+      if (value || isBool) {
```

Comment:
if the default is false, couldn't this also have been solved with something like `value !== undefined`?